README.md

```diff
@@ -176,32 +176,48 @@ The most relevant arguments are the following:
 
 ### Example Commands for Testing the Driver
 
-- To start the robot driver and controllers, use:
-   ```
-   ros2 launch ur_bringup ur_control.launch.py ur_type:=ur5e robot_ip:=yyy.yyy.yyy.yyy use_fake_hardware:=true launch_rviz:=true
+Allowed UR-Type strings: `ur3`, `ur3e`, `ur5`, `ur5e`, `ur10`, `ur10e`, `ur16e`.
 
-   (then start the external_control URCap program from the pendant, as described above)
-   ```
-   For an offline test with the emulated hardware you can just copy-paste this line.
-   To run on the hardware, write the IP address of your robot and omit the `use_fake_hardware` argument.
+##### 1. Start hardware, simulator or mockup
 
-   **NOTE**: If controllers are not starting automatically, i.e., the robot state is not shown in rviz, you can start them manually:
-   ```
-   ros2 control load_controller --set-state start joint_state_broadcaster
-   ros2 control load_controller --set-state start joint_trajectory_controller
-   ```
+- To do test with hardware, use:
+  ```
+  ros2 launch ur_bringup ur_control.launch.py ur_type:=<UR_TYPE> robot_ip:=<IP_OF_THE_ROBOT> launch_rviz:=true
+  ```
+  For more details check the argument documentation with `ros2 launch ur_bringup ur_control.launch.py --show-arguments`
+
+  After starting the launch file start the external_control URCap program from the pendant, as described above.
 
-   To check the controllers' state use `ros2 control list_controllers` command.
+- To do an offline test with URSim check details about it in [this section](#usage-with-official-ur-simulator)
+
+- To use mocked hardware (capability of ros2_control), use `use_fake_hardware` argument, like:
+  ```
+  ros2 launch ur_bringup ur_control.launch.py ur_type:=ur5e robot_ip:=yyy.yyy.yyy.yyy use_fake_hardware:=true launch_rviz:=true
+  ```
+
+  **NOTE**: Instead of using global launch file for control stack, there are also prepeared launch files for each type of UR robots named. The accept the same arguments are the global one and are used by:
+  ```
+  ros2 launch ur_bringup <ur_type>.launch.py
+  ```
+
+##### 2. Sending commands to controllers
+
+**NOTE**: If controllers are not starting automatically, i.e., the robot state is not shown in rviz, you can start them manually:
```

Comment:
If they do not get started automatically, I would assume that there is something severely wrong, right? So, trying to start the `joint_state_broadcaster` in that case would probably only lead to more error output.

Reply:
Probably yes. This was an old solution to an old issue. Thinking more about it, you are right. This part is obsolete...